static/src/js/components/ChunkedOrderModal/__tests__/ChunkedOrderModal.test.js

```diff
@@ -58,6 +58,17 @@ describe('ChunkedOrderModal component', () => {
     expect(message).toEqual('Orders for data containing more than 2,000 granules will be split into multiple orders. You will receive a set of emails for each order placed.')
   })
 
+  test.skip('should render collection specific instructions', () => {
+    const { enzymeWrapper } = setup()
+    // TODO this doesn't display the actually information, just [object Object]
```

Comment:
Did you mean to keep this? If so, we should add a ticket to track the improvement.